script/Ciel/Day 10/10_01_CIEL10_5.txt

```diff
@@ -872,72 +985,82 @@ That's the kind of human I am, and I'd like you to be fully aware of it."
 [sha:9262a4b0d86c81ebfd4ce2aabf32c7885059da70]{
 -- Page 348, Offset 28010.
 -- 先輩はいつもの笑顔で、他人事のように自分の事を語って聞かせる。
-With her usual smile, Senpai tells me about herself like it doesn't concern her.
+With her usual smile on her face,
+ Senpai tells me about herself detachedly, as though she's describing someone else.
 }
 [sha:588fc37948cdc651965a7773c5ba6d5c9be4222a]{
 -- Page 348, Offset 28011.
 -- それが冗談まじりの自嘲なのか、本気でそう思っているのか、俺には分からない。
-Is she sardonically making fun of herself, or is it really what she believes? I don't know.
+Is it her way of being jokingly self-derisive,
+ or is that really how she sees herself?
+ I can't tell for sure.
 }
 [sha:fea365d56664ddd96d8b33251f6902b591aaf250]{
 -- Page 348, Offset 28012.
 -- ただ―――どうしても引っかかるものがあった。
-Still――― something nevertheless caught my attention.
+Either way, this raises a question that I have to know the answer to.
 }
 [sha:d30b3f1286b4f1df49bcf3bfea22e938253a62fc]{
 -- Page 349, Offset 28013.
 -- 「……でも、傷を負えば痛いじゃないですか。生き返るから怖くないとしても、痛みはあるでしょう？
-"But... being injured still hurts, right? Even if you're not afraid since you'll revive, you still feel pain, don't you?
+"Even so, it still hurts when you get injured, right?
+ You may not be afraid because you know you'll come back to life, but you still feel pain, don't you?
 }
 [sha:58328e214c8bb3dc60ecd0ee04a72cd7d2c4f7aa]{
 -- Page 349, Offset 28014.
 -- ならやっぱり先輩は、」
-If so, then you really are br―――"
+If so, then you really are―――"
 }
 [sha:a4459c71b18bf83cb4e65fca16641acd70697299]{
 -- Page 350, Offset 28015.
 -- 「そんなの、<痒|かゆ>かったり、イヤだったりするだけです。
-"That's nothing more than an inconvenience, like an insect bite.
+"That's nothing more than an inconvenience, a mild itch.
 }
 [sha:732de6e468d895f0115795a92b5f4881fa6e7f8e]{
 -- Page 350, Offset 28016.
 -- 蘇生が約束されている私に恐怖はありません。ただ痛いだけなら誰にだって耐えられるでしょう？」
-There is no fear in me, to whom resurrection has been promised. Anyone should be able to endure it if it simply hurts and nothing more, right?"
+As someone who has been promised resurrection, I feel no fear.
+ Anyone would be able to bear death if all it took was a little pain, right?"
 }
 [sha:64588856376ee8efa0ffa3a6d86c91bf5230219c]{
 -- Page 351, Offset 28017.
 -- 「……そう、誰だって私と同じ体だったのなら、同じ事ができたんです。
-"Yes... Anyone could have done the same if they had the same body I have.
+"Yes... Anyone would be capable of the same things I am if they had a body like this.
 }
 [sha:b56ba873644e798b9af12f51bb4fbf4fa89958d9]{
 -- Page 351, Offset 28018.
 -- ……昔、私はこの体になるきっかけで罪を犯しました。
-A long time ago, as I came into possession of this body... I have sinned.
+When, many years ago, I came into possession of this body... I committed a sin.
 }
 [sha:f998aa4f29b225cdcfb38c10d801704323fc5450]{
 -- Page 351, Offset 28019.
 -- 多くの間違いをおかして、多くの人命を見殺しにした。
-I have made many mistakes and I have helplessly witnessed the death of many people.
+I made a great many mistakes,
+ and caused the deaths of countless people.
```

Comment:
Feel like the commit collocation is more suited to this occasion. Also, I think the repetition here is lyrical enough.
>I committed a great many mistakes,
>and so allowed a great many people to die.

Reply:
Commit used as collocation with sin in prev line, didn't want to dupe.

Reply:
Well then